### ISSUE TITLE
read.argo() not getting PRES_QC for sample file

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1242]
I don't know whether this is a problem in our code or in this particular data file.  (I ran into this when I was looking more carefully at the output from `devtools::test()`.)

```R
> library(oce)
> d<-read.oce("/Users/kelley/git/oce/tests/testthat/local_data/BR5904179_001.nc")
[1] "vobjtovarid4: error #F: I could not find the requsted var (or dimvar) in the file!"
[1] "var (or dimvar) name: PRES_QC"
[1] "file name: /Users/kelley/git/oce/tests/testthat/local_data/BR5904179_001.nc"
/Users/kelley/git/oce/tests/testthat/local_data/BR5904179_001.nc has no variable named 'PRES_QC'
```

**Comment by dankelley** [comment]
I will take notes in this comment, editing it as I learn more. The point of taking notes in this way is to avoid driving other devels crazy with email notifications as I whittle my way to the core of this problem.
```R
> ncvar_get(f, "STATION_PARAMETERS")
      [,1]                                                              
 [1,] "PRES                                                            "
 [2,] "                                                                "
 [3,] "                                                                "
 [4,] "                                                                "
 [5,] "                                                                "
 [6,] "                                                                "
 [7,] "                                                                "
 [8,] "                                                                "
 [9,] "                                                                "
[10,] "                                                                "
[11,] "                                                                "
      [,2]                                                              
 [1,] "PRES                                                            "
 [2,] "TEMP_DOXY                                                       "
 [3,] "BPHASE_DOXY                                                     "
 [4,] "DOXY                                                            "
 [5,] "FLUORESCENCE_CHLA                                               "
 [6,] "CHLA                                                            "
 [7,] "BETA_BACKSCATTERING700                                          "
 [8,] "BBP700                                                          "
 [9,] "UV_INTENSITY_DARK_NITRATE                                       "
[10,] "UV_INTENSITY_NITRATE                                            "
[11,] "NITRATE                                                         "
```

**Comment by dankelley** [comment]
I've looked into this enough to know that the warning message is sensible -- the file does *not* contain `PRES_QC`. I don't know why it lacks this data item, but I don't care very much, either, since it's an odd sort of file (e.g. no salinity) that is from a source that I've not recorded ... maybe the file has been updated, for all I know!

I cleaned up the warning so that it shows only the oce-level message (i.e. discarding the portions that were starting with `[1]`, seen in the first item in this issue report).

Also, I've cleaned up the test suite so that this known warning is stated as being expected. This is the last of 15 `devtools::test()` messages that I've cleared today, finishing the list ... now, the test is clean, with no errors or warnings.
